lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -547,11 +549,12 @@ fn do_test_monitor_temporary_update_fail(disconnect_count: usize) {
 	let events_5 = nodes[1].node.get_and_clear_pending_events();
 	assert_eq!(events_5.len(), 1);
 	match events_5[0] {
-		Event::PaymentClaimable { ref payment_hash, ref purpose, amount_msat, receiver_node_id, via_channel_id, .. } => {
+		Event::PaymentClaimable { ref payment_hash, ref purpose, amount_msat, receiver_node_id, ref inbound_channel_ids, .. } => {
 			assert_eq!(payment_hash_2, *payment_hash);
 			assert_eq!(amount_msat, 1_000_000);
 			assert_eq!(receiver_node_id.unwrap(), nodes[1].node.get_our_node_id());
-			assert_eq!(via_channel_id, Some(channel_id));
+			assert_eq!(inbound_channel_ids.len(), 1);
```

Comment:
Redundant now?

Reply:
Thanks for catching it! ✨
Looks like I forgot to remove this, when I update the code.
Fixed in [pr3655.13](https://github.com/shaavan/rust-lightning/commits/pr3655.13)